students/thorn/lesson02/series.py

```diff
@@ -0,0 +1,82 @@
+'''
+Thomas Horn
+The following definitions are used to computer the Fibonacci and Lucas Series
+throughout this program.
+    Fibonacci - starts with 0, 1 and sums the previous 2 numbers
+    Lucas - starts with 1, 2 and sums the previous 2 numbers
+'''
+
+def fibonacci(n):
+    """ Returns the nth value in the fibonacci sequence. """
+    if n <= 1:
+        return 0
```

Comment:
Remember that the function should return the nth value in the fibonacci series (starting with zero index), so we know that `fibonacci(0) == 0` and `fibonacci(1) == 1` so instead of
```
    if n <= 1:
        return 0
```
this should be 
```
    if n == 0:
        return 0
    if n == 1:
        return 1
```
Other than that, looks good!